trieste/acquisition/rule.py

```diff
@@ -1068,3 +1081,118 @@ def state_func(
             return state_, points
 
         return state_func
+
+
+class BatchHypervolumeSharpeRatioIndicator(
+    AcquisitionRule[TensorType, SearchSpaceType, ProbabilisticModelType]
```

Comment:
For now, you can change this (and the other occurrences) of `ProbabilisticModelType` to `ProbabilisticModel`. The former is a type variable that lets us restrict rules to support only certain types of model. This rule can (currently) support any model (just like ProbabilityOfImprovement). If in future we allow users to specify e.g other builders then we'll need to make this a type variable again, so that they can pass e.g. a builder that only supports certain types of model (so long as they only use the rule with that type of model).

```suggestion
    AcquisitionRule[TensorType, SearchSpaceType, ProbabilisticModel]
```

Reply:
(remember to change this to ProbabilisticModel. You can always change it back to ProbabilisticModelType later if you need to when you make the rule more general)